engineio/benches/engineio.rs

```diff
@@ -0,0 +1,206 @@
+use bytes::Bytes;
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use native_tls::Certificate;
+use native_tls::TlsConnector;
+use rust_engineio::error::Error;
+use rust_engineio::{
+    packet::{Packet, PacketId},
```

Comment:
Would it make sense to call this in a ci job?

Reply:
Perhaps on tag creation? Takes a bit to run, wouldn't want it to block all PRs.

Reply:
Ah yeah, tag creation sounds good! But lets first merge this PR, then we can merge it.